.drone.yml

```diff
@@ -66,3 +66,23 @@ steps:
       event:
         include:
         - push
+
+  - name: docker-build--system-info
+    image: plugins/docker
+    settings:
+      repo: rancher/system-info
```

Comment:
Should this be systems-info, not system-info? The references in the chart use systems-info.

Reply:
Fixed